src/script/components/app-header.ts

```diff
@@ -165,7 +165,8 @@ export class AppHeader extends LitElement {
   }
   
   recordStep(text: string){
-    recordProcessStep('pwa-builder', `${text}-clicked`, AnalyticsBehavior.ProcessCheckpoint);
+    let pageName = window.location.pathname.slice(1);
```

Comment:
You should move this function to the analytics file instead of redefining it here. Consider also making a version of the recordProcessStep function that already includes the 'pwa-builder' process name since it is always going to be the same (instead of having to specify it each time) - basicly factor out the process name for all into a single variable

Reply:
i originally thought to do this but for some reason i didn't. I'll take care of it.